qutebrowser/javascript/webengine_caret.js

```diff
@@ -0,0 +1,1104 @@
+/**
+ * Copyright 2016-2017 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+ *
+ * This file is part of qutebrowser.
+ *
+ * qutebrowser is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * qutebrowser is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * Ported chrome-caretbrowsing extension.
+ * Create and control div caret, which listen commands from qutebrowser,
+ * change document selection model and div caret position.
+ */
+
+"use strict"; // eslint-disable-line max-lines
```

Comment:
Wouldn't it make more sense to disable some eslint things (`max-lines`, `max-statements`, `max-len`, `complexity`, `max-params`, maybe more?) globally for this file instead of per-line?